configure.ac

```diff
@@ -760,13 +760,13 @@ linux*)
   AC_CHECK_PROG(USE_HEAD, head, yes, no)
   if test $USE_LDD == "yes" -a $USE_HEAD
   then
-    if test "`ldd --version | head -1 | awk '{print $4>=2.23?"1":"0"}' 2>/dev/null`" = "1"
+    if test "`ldd --version | head -1 | awk '{print ($4>=2.23)?"1":"0"}' 2>/dev/null`" = "1"
     then
       AC_MSG_NOTICE([glibc version >= 2.23. glibc AVX-512 optimisation supported.])
```

Comment:
This came to our internal review and many users misunderstand about the AVX512 multibuffer optimization. So it is better we add it for glibc AVX512 memcpy optimization in this message.

Can we add that as part of this  commit  ? and include it in the upcoming release ?